model/Course.js

```diff
@@ -26,6 +26,7 @@ const courseSchema = new Schema({
   wi: { type: Boolean, default: false },
   taken: { type: Boolean, default: false },
   preReq: { type: Array },
+  postReq: { type: Array, default: [] },
```

Comment:
Why do we need to note postReq for Course too? Why not just SISCourseV?

Reply:
Will double check this, but I thought that when a course was already in a plan, and a user clicks on that course, the popup uses the user course object to display info(and thus needs the post-req info to display that tab)